app/server/sonicpi/lib/sonicpi/osc/clients.rb

```diff
@@ -0,0 +1,35 @@
+module SonicPi
+  module OSC
+    class Clients
+
+      def initialize(client_class)
+        @client_class = client_class
+        @clients = []
+        @keep_open = []
+      end
+
+      def new_client(port, keep_open=false)
+        client = @client_class.new(
+          "127.0.0.1", port, use_encode_cache: true)
```

Comment:
Do the parameters here need to match those removed from sonic-pi-server.rb lines 49 & 51 above? If so, I think this needs to say ```use_encoder_cache: true```